Prose/ProseLib/Sources/App/AppReducer.swift

```diff
@@ -24,6 +28,30 @@ public let appReducer: Reducer<
         action: /AppAction.main,
         environment: \AppEnvironment.main
     ),
+    Reducer { state, action, environment in
+        switch action {
+        case let .didLogIn(jid):
+            state.route = .main(MainScreenState(
+                sidebar: .init(credentials: .init(jid: jid))
+            ))
+
+        case let .auth(.loginResult(.success(jid))):
+            let url = URL(staticString: "prose://main")
+            return environment.openURL(url, .init())
+                .receive(on: environment.mainQueue)
+                .map { AppAction.didLogIn(jid: jid) }
+                .mapError(EquatableError.init)
+                .catch { error -> AnyPublisher<AppAction, Never> in
+                    fatalError("Failed to open URL <\(url.absoluteString)>: \(error.localizedDescription)")
+                }
+                .eraseToEffect()
+
+        default:
```

Comment:
Unfortunately this slipped through. Please no `default` cases. We want the compiler to help us by checking for exhaustiveness. Could you please fix this in #38?